doc/api/vm.md

```diff
@@ -805,12 +805,6 @@ const vm = require('vm');
 ## `vm.compileFunction(code[, params[, options]])`
 <!-- YAML
 added: v10.10.0
-changes:
-  - version:
-    - v14.1.0
-    - v13.14.0
-    pr-url: https://github.com/nodejs/node/pull/32985
-    description: The `importModuleDynamically` option is now supported.
```

Comment:
We should probably keep this and add a new entry for the version that reverted it

Reply:
Done, PTAL